creator/Reference/Content/EntityReference/Examples/EntityAttributes/minecraftAttribute_spell_effects.md

```diff
@@ -7,43 +7,60 @@ ms.prod: gaming
 
 ## Parameters
 
-|Name |Default Value  |Type  |Description  |
-|:----------|:----------|:----------|:----------|
-|add_effects|*not set* | List|  List of effects to add to this entity after adding this component |
-|remove_effects|*not set* | String|  List of identifiers of effects to be removed from this entity after adding this component |
+### `add_effects`
 
-### add_effects
+`add_effects` is a list variable that can use status effect objects to define which effects should be added to the entity when the component is applied. Each object contains the following properties:
 
-`add_effects` is a list variable that can use the following parameters:
+* `effect`: The string identifier of the status effect to add. These are the same as used in the `/effect` command.
+* `duration`: The amount of time in seconds the effect should last. This allows for fractional numbers. For example, instant effects should be set to 0.05 seconds (one tick).
+* `amplifier`: The level of the effect, same as used in the `/effect` command (0 for level I, 1 for level II, etc). Defaults to 0.
+* `ambient`: Boolean value that should cause the particles emitted by the entity to be partially transparent. This does not work properly, resulting in this property having no effect. Defaults to false.
+* `visible`: Boolean value that determines whether the particles emitted by the entity should appear. Defaults to true.
+* `display_on_screen_animation`: Boolean value. When set to true, applying this effect displays an animated graphic on-screen similar to the totem of undying effect. Obviously, this only works for players. Defaults to false.
 
-| effect| *not set*| String|  Effect to add to this entity. Includes 'duration' in seconds, 'amplifier' level, 'ambient' if the effect should be partially transparent, and 'visible' if the effect should be visible |
+### `remove_effects`
+This can either be a single string identifier for a status effect, or a list of them. These define which effects should be removed from the entity, if present, when the component is applied.
 
 ## Example
 
 ```json
-"minecraft:spell_effects":{
-    "add_effects":[
-    {
-        "effect": "instant_health",
-        "duration": 25
-    }
+"minecraft:spell_effects": {
+    "add_effects": [
+        {
+            "effect": "regeneration",
+            "duration": 25,
+            "amplifier": 2,
+            "visible": false,
+            "display_on_screen_animation": true
+        },
+        {
+            "effect": "instant_health",
+            "duration": 0.05
+        }
     ],
-    "remove_effects": ["poison", "instant_health"]
+    "remove_effects": ["poison", "wither"]
 }
 ```
 
 ## Vanilla entities examples
 
-### zombie_villager_v2
+### `zombie_villager_v2`
+
+Note that the use of `heal` in this example is incorrect and does nothing, since `instant_health` is the correct name for that effect.
```

Comment:
So while `heal` doesn't do anything to the player, it doesn't actually do nothing - this is an effect that is used to transform a zombie villager into villager using a weakness potion and a golden apple. I will look to see if there is a better snippet to use as an example.

Reply:
I'm pretty sure the `transformation` component does that. In fact, just now I tested removing the `spell_effects` component from zombie villagers altogether and the conversion to villager with weakness + apple was still successful.

Reply:
If we can definitively prove that `heal` does something, we should definitely explicitly document it as a blatant exception to the purpose of this component (applying `/effect` effects).